Source/Additional Samples/GnssTracker_SQLite_Demo/Controllers/DisplayController.cs

```diff
@@ -0,0 +1,148 @@
+﻿using GnssTracker_SQLite_Demo.Models.Logical;
+using Meadow;
+using Meadow.Foundation;
+using Meadow.Foundation.Graphics;
+using Meadow.Logging;
+using SimpleJpegDecoder;
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace GnssTracker_SQLite_Demo.Controllers
+{
+    public static class DisplayController
+    {
+        private static int counter = 0;
+        private static Logger Log { get => Resolver.Log; }
+        private static MicroGraphics graphics { get; set; }
+
+        private static Font12x20 largeFont { get; set; }
+        private static Font4x8 smallFont { get; set; }
+
+        private static bool rendering { get; set; }
+        private static object renderLock = new object();
+
+        public static void Initialize(IGraphicsDisplay display)
+        {
+            largeFont = new Font12x20();
+            smallFont = new Font4x8();
+
+            graphics = new MicroGraphics(display)
+            {
+                Rotation = RotationType._270Degrees,
+                Stroke = 1
+            };
+
+            ShowSplashScreen();
+        }
+
+        private static void ShowSplashScreen()
+        {
+            graphics.Clear(Color.White);
+
+            DrawJPG(0, 0, "gnss_tracker.jpg");
+            graphics.DrawRectangle(5, 5, 240, 112, Color.Black);
+
+            graphics.Show();
+        }
+
+        private static byte[] LoadResource(string filename)
+        {
+            var assembly = Assembly.GetExecutingAssembly();
+            var resourceName = $"GnssTracker_SQLite_Demo.{filename}";
+
+            using (Stream stream = assembly.GetManifestResourceStream(resourceName))
+            {
+                using (var ms = new MemoryStream())
+                {
+                    stream.CopyTo(ms);
+                    return ms.ToArray();
+                }
+            }
+        }
+
+        private static void DrawJPG(int x, int y, string filename)
```

Comment:
Ideally use the MicroGraphics buffers instead of drawing by hand - you can see an example in the DefCon badge project